eclair-core/src/main/scala/fr/acinq/eclair/blockchain/bitcoind/ZmqWatcher.scala

```diff
@@ -385,25 +386,14 @@ private class ZmqWatcher(chainHash: ByteVector32, blockCount: AtomicLong, client
   }
 
   def checkUtxos(): Future[Unit] = {
```

Comment:
I think this whole check should be removed from the `ZmqWatcher` and instead move to the `BalanceActor`, which should check our utxos at every block to update the metric. I always had a plan to remove it from here once we had some actor dedicated to monitoring our utxos/balance.